flink-autoscaler/pom.xml

```diff
@@ -31,6 +31,13 @@ under the License.
     <name>Flink Autoscaler</name>
     <packaging>jar</packaging>
 
+    <properties>
+        <surefire.module.config> <!-- required by
+        JobTopologyTest
+        -->--add-opens=java.base/java.lang=ALL-UNNAMED
+        </surefire.module.config>
```

Comment:
```suggestion
        <!-- required by JobTopologyTest -->
        <surefire.module.config>--add-opens=java.base/java.lang=ALL-UNNAMED</surefire.module.config>
```

How about this? It's easy to read.

Reply:
in case of just one flag it probably makes sense